crates/aptos-crypto/src/multi_ed25519.rs

```diff
@@ -116,6 +117,75 @@ impl MultiEd25519PublicKey {
     pub fn to_bytes(&self) -> Vec<u8> {
         to_bytes(&self.public_keys, self.threshold)
     }
+
+    /// Like `validate_bytes_count_checks`, but only returns the number of sub PKs that passed
+    /// the validation criteria (i.e., deserialized correctly and were not in a small subgroup).
+    pub fn validate_bytes(bytes: &[u8]) -> bool {
```

Comment:
Not returning what's described in the function comment?

Reply:
And no one is calling it. by design?

Reply:
Removed it.